components/list_item/default_list_item.vue

```diff
@@ -10,7 +10,8 @@
     </section>
     <section class="d-fl-grow1 d-of-hidden">
       <div v-if="$slots.default">
-        <!-- @slot Slot for the main content -->
+        <!-- @slot S
+          lot for the main content -->
```

Comment:
Weird, what happened here? Linter do this?

Reply:
I don't know. I'll correct and see if it does it again, though I can't see why it would; I'd bet on an errant keystroke.